src/common/metrics/case_density.tsx

```diff
@@ -61,19 +63,39 @@ export const CASE_DENSITY_LEVEL_INFO_MAP: LevelInfoMap = {
 };
 
 function renderStatus(projections: Projections): React.ReactElement {
-  const { totalPopulation, currentDailyAverageCases } = projections.primary;
+  const {
+    totalPopulation,
+    currentDailyAverageCases,
+    region,
+  } = projections.primary;
   const currentCaseDensity = projections.getMetricValue(Metric.CASE_DENSITY);
   const locationName = projections.locationName;
+  const isBlocked = getRegionMetricOverride(region, Metric.CASE_DENSITY)
```

Comment:
This common logic between the metric pages could/should maybe be abstracted out, but I don't think it's worth worrying about at this point